static/src/js/components/AdminPreferencesMetrics/AdminPreferencesMetrics.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Row, Col } from 'react-bootstrap'
+
+import { AdminPage } from '../AdminPage/AdminPage'
+import { AdminPreferencesMetricsList } from './AdminPreferencesMetricsList'
+
+export const AdminPreferencesMetrics = ({
+  onFetchAdminMetricsPreferences,
+  metricsPreferences
+}) => {
+  useEffect(async () => {
+    await onFetchAdminMetricsPreferences()
```

Comment:
Fetching should be done within the container component, not the presentation component